README.md

```diff
@@ -1,128 +1,276 @@
 <p align="center">
-<img
-    src="https://neo-cdn.azureedge.net/images/neo_logo.svg"
-    width="250px">
+  <a href="https://neo.org/">
+      <img
+      src="https://neo3.azureedge.net/images/logo%20files-dark.svg"
+      width="250px" alt="neo-logo">
+  </a>
 </p>
 
+<h3 align="center">Neo Blockchain</h3>
+
 <p align="center">
-  <a href="https://travis-ci.org/neo-project/neo">
-    <img src="https://travis-ci.org/neo-project/neo.svg?branch=master" alt="Current TravisCI build status.">
+   A modern distributed network for the Smart Economy.
+  <br>
+  <a href="https://docs.neo.org/docs/en-us/index.html"><strong>Documentation »</strong></a>
+  <br>
+  <br>
+  <a href="https://github.com/neo-project/neo"><strong>Neo</strong></a>
+  ·
+  <a href="https://github.com/neo-project/neo-vm">Neo VM</a>
+  ·
+  <a href="https://github.com/neo-project/neo-plugins">Neo Plugins</a>
+  ·
+  <a href="https://github.com/neo-project/neo-devpack-dotnet">Neo DevPack</a>
+  ·
+  <a href="https://github.com/neo-project/neo-cli">Neo CLI</a>
+</p>
+<p align="center">
+  <a href="https://twitter.com/neo_blockchain">
+      <img
+      src=".github/images/twitter-logo.png"
+      width="25px">
   </a>
-  <a href="https://github.com/neo-project/neo/releases">
-    <img src="https://badge.fury.io/gh/neo-project%2Fneo.svg" alt="Current neo version.">
+  &nbsp;
+  <a href="https://medium.com/neo-smart-economy">
```

Comment:
Hi, Rick. I am not sure if these social media links are appropriate here. Because this is a repo for Neo core code, we think we just need to write something highly related how to use neo.

Reply:
Hi @wanglongfei88 , they need a way to explore the ecosystem. The developers should return here several times, this way we ensure he can get all the info that he might need (and it is a very small icon)

Reply:
I agree developers may return here several times. But does it mean that we need to provide everything in Neo ecosystem here ? I am not sure it does. That is just my opinion.